addons/l10n_it_edi/tests/test_edi_export.py

```diff
@@ -304,6 +304,29 @@ def setUpClass(cls):
         cls.zero_tax_invoice._post()
         cls.negative_price_invoice._post()
         cls.negative_price_credit_note._post()
+    
+    
+    def test_export_zero_amount_move(self):
+        """ When a move has an amount of 0, a float division by zero error is triggered.
+        """
```

Comment:
```suggestion
        """When a move has an amount of 0, a float division by zero error is triggered."""
```
No need for a multi-line doc string (see [here](https://peps.python.org/pep-0257/#one-line-docstrings)).